### ISSUE TITLE
Exporting Bookmarks Then Importing Not Happening As Expected... 

### ISSUE THREAD

**Issue by ghost** [QupZilla/qupzilla#1644]
Hi,

After arranging the bookmarks the way one likes, and then exporting them to HTML file, and then importing them, I'm expecting them to appear as they were arranged, but this is not happening. :(

Here's a screen shot showing them arranged how I like them;

![q1](https://cloud.githubusercontent.com/assets/11289036/7224407/43c61632-e721-11e4-8700-d35bbffe99da.jpg)

Now I imported like how they look above to an HTML file, then I removed the ~/.config/qupzilla directory, so QupZilla is being started for the first time, and I import these bookmarks, but they appear just as they always do, as some sort of default layout they're always given. :(

![q2](https://cloud.githubusercontent.com/assets/11289036/7224417/83101f0e-e721-11e4-977e-784e5bd78c5b.jpg)

I hope this will please be fixed soon, so they can be exported how they look, and then imported back to this layout.

This was tested on QupZilla 1.8.6 with Qt4 & Qt5 in Linux...

thank you

**Comment by ff2000** [comment]
AFAIK that's intended behaviour. Importing bookmarks from another browser should never f*** up the bookmarks from the importing browser (here: QupZilla). You can easily resort your Bookmarks via DnD, btw :)

**Comment by meuwe** [comment]
I'm not importing them from another browser, I originally imported them from Firefox, but then had them end up like this the first time, so then in the first screen after arranging them the way I like, I then exported them from QupZilla.

Now that I've created the layout I like, and then exported them from QupZilla, I'd expect they should maintain this layout, otherwise what's the point, especially when you have a lot of bookmarks and spend all the time fixing them, only to have to repeat the process everytime... :(

DnD? 

By the way I just test this by copying the ```bookmarks.json``` to a new ~/.config/qupzilla directory and the bookmakrs were all back the way I had them... This is how the exporting and importing of the bookmarks should be work. :)
